File: API/Features/Registrars/Dtos/RegistrarReadDto.cs
using API.Infrastructure.Classes;

namespace API.Features.Registrars {

    public class RegistrarReadDto {

        public int Id { get; set; }
        public SimpleEntity Ship { get; set; }
        public string Fullname { get; set; }
        public string Phones { get; set; }
        public string Email { get; set; }
        public string Fax { get; set; }
        public string Address { get; set; }
        public bool IsPrimary { get; set; }
        public bool IsActive { get; set; }
        public string UserId { get; set; }

    }

}

File: FrontEnd/src/app/features/registrars/classes/dtos/registrar-read-dto.ts
import { Metadata } from 'src/app/shared/classes/metadata'
import { ShipAutoCompleteVM } from '../../../ships/classes/view-models/ship-autocomplete-vm'

export interface RegistrarReadDto extends Metadata {

    id: number
    ship: ShipAutoCompleteVM
    fullname: string
    phones: string
    email: string
    fax: string
    address: string
    isPrimary: boolean
    isActive: boolean
    user: string
    lastUpdate: string

}
